src/mqt/bench/evaluation/evaluation_visualization.ipynb

```diff
@@ -0,0 +1,300 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2d493d7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pickle\n",
+    "from pathlib import Path\n",
+    "import matplotlib\n",
+    "\n",
+    "font = {\"size\": 12}\n",
+    "matplotlib.rc(\"font\", **font)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12d027e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with Path(\"./evaluation_data.pkl\").open(\"rb\") as f:\n",
+    "    eval_data = pickle.load(f)\n",
```

Comment:
Already stated it above, but would it be too time consuming to perform the calculation of the evaluation data every time?
That would avoid having to store these pickle files.